packages/twenty-server/src/engine/integrations/serverless/drivers/local.driver.ts

```diff
@@ -39,22 +41,40 @@ export class LocalDriver
     this.fileStorageService = options.fileStorageService;
   }
 
+  private getInMemoryLayerFolderPath = (version: number) => {
+    return join(SERVERLESS_TMPDIR_FOLDER, COMMON_LAYER_NAME, `${version}`);
+  };
+
+  private async createLayerIfNotExists(version: number) {
```

Comment:
@martmull I would like to discuss this with you. A layer is not really specific enough to me. When we say a lambda layer, we know we are talking about modules management. But I am not sure this naming should be used for the local driver. 

I feel we need something more generic there, related to modules management.

Reply:
I used layer to match the aws naming.
For example, when we will use python serverless functions, the "module" naming won't fit as it is called "requirements" in python